README.md

```diff
@@ -787,12 +787,20 @@ Alpine Linux, unlike mainstream/traditional Linux distributions, is based on [Bu
 
 There is a `-s` flag for `nvm install` which requests nvm download Node source and compile it locally.
 
-If installing nvm on Alpine Linux *is* still what you want or need to do, you should be able to achieve this by running the following from you Alpine Linux shell:
+If installing nvm on Alpine Linux *is* still what you want or need to do, you should be able to achieve this by running the following from you Alpine Linux shell, depending on which version you are using:
 
+### Alpine Linux 3.13+
+```sh
+apk add -U curl bash ca-certificates openssl ncurses coreutils python3 make gcc g++ libgcc linux-headers grep util-linux binutils findutils
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+```
+
+### Alpine Linux 3.5 - 3.12
 ```sh
 apk add -U curl bash ca-certificates openssl ncurses coreutils python2 make gcc g++ libgcc linux-headers grep util-linux binutils findutils
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
 ```
+_Note: Alpine Linux 3.5 - 3.12 may install older versions of NodeJS, since these versions don't support the LTS version(s) of NodeJS, which require the package `python3` to successfully build, whereas Alpine 3.13+ does support it. For the best results, please use Alpine Linux 3.13+._
```

Comment:
“LTS” is a moving target, so that doesn’t help - can you be specific about which version numbers work with 3.13+, and which version numbers require 3.12? Let’s also avoid the phrase “best results” since that definition depends on the desired node version to be used :-)

Reply:
I used the wording of LTS, for the reason that it moves, so we don't have to update it for every LTS version released, as well as it is Long Term Support, so it shouldn't change for virtually a while. I can still remove the wording, "best results" though.

Reply:
Right, but the node versions that require alpine < 3.13 are static and won’t ever change.

Reply:
Which is why, I mentioned the older versions of Alpine don't support LTS. I will change it still, if you are set on it.

Reply:
I updated it, how does it look now?